ui/app/common/attributeTypes/directives/attributeTypes.js

```diff
@@ -30,6 +40,27 @@ angular.module('bahmni.common.attributeTypes', []).directive('attributeTypes', [
                 });
                 attributeValueConceptType.value = concept && concept.fullySpecifiedName;
             };
+
+
+            $scope.suggest = function (string) {
+
```

Comment:
Remove this blank line. Please run grunt in your `ui` folder and see if any more problems come up